pytest.ini

```diff
@@ -0,0 +1,8 @@
+[pytest]
+filterwarnings =
+    ignore:.*'soft_unicode' has been renamed to 'soft_str'*:DeprecationWarning
+    ignore:unclosed file .*:ResourceWarning
+env_files =
+    test.env
+testpaths =
+    tests
```

Comment:
Do we need this file?

Reply:
I believe we had pulled this over as part of the tests fixes yesterday can test to see if it can be removed.